Templates/Tests/TemplatesTests.swift

```diff
@@ -32,14 +37,18 @@ class TemplatesTests: QuickSpec {
                 print(error)
             }
         }
-
+        #if canImport(ObjectiveC)
+        let contextSources = "\(resources)/Context"
+        #else
+        let contextSources = "\(resources)/Context_Linux"
```

Comment:
Different file folders are used for two reasons:

1. SPM does not have a method to specify where to put the copied resources (`Resource.copy(path:locatization:)`). 
3. Because of the [crash in Swift compiler](https://github.com/apple/swift/issues/68296), `AutoCodable.swift` file needs to be omitted when tests are run on Linux